typescript/infra/package.json

```diff
@@ -82,7 +82,7 @@
     "test:unit": "mocha --config ../sdk/.mocharc.json \"test/**/*.test.ts\" --ignore \"test/{warp-configs,balance-alerts}.test.ts\"",
     "test:hardhat": "yarn hardhat-esm test test/govern.hardhat-test.ts",
     "test": "yarn test:unit && yarn test:hardhat && yarn test:warp && yarn test:balance",
-    "test:ci": "yarn test:unit && yarn test:hardhat",
+    "test:ci": "yarn test:unit -e mainnet3 && yarn test:hardhat",
```

Comment:
@paulbalaji i can't figure out how the tests would have run without this - is that crazy?

Reply:
i think it's because this is the first PR to add a test that references a script like this

https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6374/files#diff-75ead07e35bf40beb87620a998ce8734fa30ec8b9931cb468ebb17dbaf04b65eR49

the `environment` is mandatory via this - but we likely just haven't got any unit tests of others

so not tooo crazy that we now require the `-e mainnet3`